### PR TITLE
rawvm doc

### PR DESCRIPTION
**Pull request by ldemailly**
Doc/notes about MySql setup on rawvm and cluster extension

(start of a) simpler demo than bookinfo

Misc: typo fix in hello world

```release-note
Cluster extension doc
```

### CODE REVIEW

**Comment by ldemailly** [comment]
I have tried the detailed setup with mariadb yet, while the steps listed do work for official mysql, will adjust/update the doc once I do but I'd like to save this for now

want to try your newly acquired "/lgtm" powers on this PR ?

**Comment by ldemailly** [comment]
@linsun please have a look. release impact: none

this is doc about mysql in our environment (and also some partial example of setup not involving bookinfo)

File: samples/rawvm/README.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+# for low volume troubleshooting:
+mysql -u root -h instance-1 --password=password
+  SET global general_log_file='/tmp/mysqlquery.log';
+  SET global general_log = 1;
+# then
+tail -f /tmp/mysqlquery.log  
+```
+
+### Or
+sudo apt-get mariadb-server
```
Comment (linsun) [inline samples/rawvm/README.md]: what is the purpose for mariadb-server?

File: samples/rawvm/README.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+Note to install rbac yaml you need:
+```
+kubectl create clusterrolebinding my-admin-access --clusterrole cluster-admin --user USERNAME
+```
+
+Then:
+
+```
+git clone https://github.com/istio/proxy.git -b rawvm-demo-0-2-2
```
Comment (linsun) [inline samples/rawvm/README.md]: do we really need this?

File: samples/rawvm/README.md (hunk 1 above)
Comment (ldemailly) [reply samples/rawvm/README.md]: It's easier to install/more 'open source'

File: samples/rawvm/README.md (hunk 2 above)
Comment (ldemailly) [reply samples/rawvm/README.md]: Some scripts are/were there. Will update later

Comment (ldemailly) [reply samples/rawvm/README.md]: Some scripts are/were there. Will update later

**Pull request merged**